applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -29,11 +29,11 @@ config BT_EXT_ADV
 
 config BT_ISO_RX_MTU
 	int
-	default 190
+	default 155
```

Comment:
Would 190 for SBC?

Reply:
Yeah, you are right. The default is 251, and I guess we could use that as well, but do you remember if that causes more issues with  scheduling, since it will have to take into account that there is a possibility that the MTU is 251?

Reply:
Yes, during the CIS channel establishment negotiation we will set QoS which contain with the maximum payload size, and large number will cause scheduler issue.
In the legacy code, we were using BT_ISO_TX_MTU as maximum payload size. I just checked the code, and now the host will use the number from the QoS setting instead of using fixed setting from BT_ISO_TX_MTU.
So probably we can set this number to 251 now.

Reply:
Nice, then I will just remove these from Kconfig.default since the zephyr-default is already 251